last_seen.py

```diff
@@ -0,0 +1,27 @@
+from brutal.core.plugin import BotPlugin, cmd, event
+from time import strftime, gmtime
+
+class LastSeen(BotPlugin):
```

Comment:
@JakubNvk the travis tests will fail if there won't be two blank lines before a class definition. Please add a blank line here. Thanks